lib/src/strings.dart

```diff
@@ -131,6 +135,27 @@ const sizemap_native_mapping = <String, int>{
 // Library imports.
 const libraryImports = 'library-imports';
 
+// Sub Keys of symbol file.
+const symbols = 'symbols';
+
+// Symbol file yaml.
+const formatVersion = "format_version";
+
+/// Current symbol file format version. This is generated when generating any
```

Comment:
https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

```dart
/// Current symbol file format version.
///
/// This is generated when generating any ...
```

Reply:
Ah, how did I miss this.